target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -187,6 +187,13 @@ case "$FIRMWARE" in
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
 		rm /lib/firmware/ath10k/QCA9888/hw2.0/board-2.bin
 		;;
+	comfast,cf-e355ac-v2)
+		caldata_extract "art" 0x5000 0x844
```

Comment:
Are you really the only device just needing 0x844 here while all other need 0x2f20?